assets/main.css

```diff
@@ -1,9 +1,6 @@
-@tailwind base;
-
 @import "./base.css";
-
-@tailwind components;
-
 @import "./components.css";
```

Comment:
Theoretically these should come after all `@tailwind` declarations. Example: https://github.com/perimetre/marathon-frontend/blob/dev/src/styles/globals.css

Reply:
(But check if this was the default of the original template or not)

Reply:
normally i would agree but this project has some custom css, so if i put tailwind first i get just 50% of the styles, and mostly a black screen, it was in fact originally like that, but even is this case i would recive a warning telling me to re-order it, so i did.